src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/OffenderController.kt

```diff
@@ -62,4 +64,45 @@ class OffenderController(private val offenderService: OffenderService, private v
     val newCom = this.comService.updateComDetails(body)
     this.offenderService.updateOffenderWithResponsibleCom(crn, newCom)
   }
+
+  @PutMapping(
+    value = ["/crn/{crn}/probation-team"],
+    produces = [MediaType.APPLICATION_JSON_VALUE]
+  )
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @Operation(
+    summary = "Updates in-flight licences associated with an offender with a new probation team.",
+    description = "Updates in-flight licences associated with an offender with a new probation team. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "The probation team was updated",
+      ),
+      ApiResponse(
+        responseCode = "400",
+        description = "Bad request, request body must be valid",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorised, requires a valid Oauth2 token",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden, requires an appropriate role",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      )
+    ]
+  )
+  @Transactional
+  fun updateProbationTeam(
+    @PathVariable crn: String,
+    @RequestBody body: UpdateProbationTeamRequest
+  ) {
+    this.offenderService.updateProbationRegion(crn, body)
+  }
```

Comment:
Just naming potentially? `updateProbationTeam` calling a service method called `updateProbationRegion`? Probation region is a specific value. Small one, not a blocker.

Reply:
Sorted now, thanks